src/Backend/Spec.ts

```diff
@@ -70,8 +74,7 @@ class BridgeSpec {
 }
 
 // TODO add more BridgeSpec like docker or ADB......
-const sdbSpec = new BridgeSpec(
-    'sdb', 'sdb devices | grep -v devices | grep device | awk \'{print $1}\'', 'sdb shell');
+const sdbSpec = new BridgeSpec('sdb', vscode.Uri.joinPath(ext?.extensionUri, 'res', 'script', 'sdbSpecList.sh').fsPath, 'sdb shell');
```

Comment:
If we change the `extensionId` in future, it would better to make this line fail by using `!`.

```suggestion
const sdbSpec = new BridgeSpec('sdb', vscode.Uri.joinPath(ext!.extensionUri, 'res', 'script', 'sdbSpecList.sh').fsPath, 'sdb shell');
```